bin-src/lib/uploadZip.js

```diff
@@ -71,7 +71,7 @@ export async function waitForUnpack(ctx, url) {
       }
     },
     {
-      retries: 30,
+      retries: 360, // 3 minutes minimum (matches the lambda timeout)
       minTimeout: 500,
       maxTimeout: 1000,
```

Comment:
We _could_ decrease this `maxTimeout` but there's some time discrepancy between upload and the lambda kicking off. This gives us some buffer for that time difference.